### ISSUE TITLE
Adding additional metadata during file upload

### ISSUE THREAD

**Issue by ds-mathias** [DS4SD/deepsearch-toolkit#185]
It would be great if there was the ability to add additional custom metadata into the Elasticsearch index during the file ingestion stage. For example, if we were uploading documents from the UK, and afterwards documents from Spain, being able to add a "country" metadata field for "UK" or "Spain" into the Elasticsearch index.

**Comment by dolfim-ibm** [comment]
This was actually released just this week, but we still have to propagate the feature to the Toolkit.

**Comment by ds-mathias** [comment]
Yes I was referring to the SDK as I couldn't find any documentation on this feature. Do you know when it will be added?
